libs/application/templates/no-debt-certificate/src/fields/ConfirmationField/index.tsx

```diff
@@ -0,0 +1,217 @@
+
+  function renderFooter() {
+    return (
+      <>
+        <Divider />
+        <Box
+          display="flex"
+          justifyContent="flexEnd"
+          paddingTop={4}
+          marginBottom={4}
+        >
+          <Button
+            icon="arrowForward"
+            iconType="outline"
+            onClick={() => {
+              window.open(`${window.location.origin}/minarsidur`, '_blank')
+            }}
+          >
+            {formatText(m.openMySites, application, formatMessage)}
+          </Button>
+        </Box>
+      </>
+    )
+  }
+  if (viewNoDebtCertificate) {
+    return (
+      <>
+        <Box
+          display="flex"
+          marginBottom={2}
+          justifyContent="spaceBetween"
+          alignItems="center"
+        >
+          <Button
+            circle
+            icon="arrowBack"
+            onClick={() => setViewNoDebtCertificate(false)}
+            colorScheme="light"
+            title="Go back"
+          />
+          <a
+            href={`data:application/pdf;base64,${document}`}
+            download="skuldleysisvottord.pdf"
+            className={styles.linkWithoutDecorations}
+          >
+            <Button icon="download" iconType="outline" variant="text">
+              {formatText(
+                m.downloadNoDebtCertificate,
+                application,
+                formatMessage,
+              )}
+            </Button>
+          </a>
+        </Box>
+        <PdfViewer file={`data:application/pdf;base64,${document}`} />
+        {renderFooter()}
+      </>
+    )
+  }
+
+  return (
+    <>
+      <Text variant="h2" marginBottom={4}>
+        {formatText(m.confirmation, application, formatMessage)}
+      </Text>
+      <Box marginBottom={3} paddingTop={0}>
+        <AlertMessage
+          type="success"
+          title={formatText(m.successTitle, application, formatMessage)}
+          message={
+            <Box component="span" display="block">
+              <Text variant="small">
+                {formatText(m.successDescription, application, formatMessage)}
+              </Text>
+            </Box>
+          }
+        />
+      </Box>
+      <Box
+        marginBottom={3}
+        marginTop={0}
+        background="blue100"
+        padding={4}
+        display="flex"
+      >
+        <LinkContext.Provider
+          value={{
+            linkRenderer: (href, children) => (
+              <a
+                style={{
+                  color: '#0061ff',
+                  textDecoration: 'none',
+                  boxShadow: 'inset 0 -1px 0 0 currentColor',
+                  paddingBottom: 4,
+                }}
+                href={href}
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                {children}
+              </a>
+            ),
+          }}
+        >
+          <Text variant="small">
+            {formatText(m.vertificationDescription, application, formatMessage)}{' '}
```

Comment:
May I suggest using this instead.
\`${formatText(m.vertificationDescription, application, formatMessage)} \`